pymongo/change_stream.py

```diff
@@ -96,7 +97,7 @@ class ChangeStream(Generic[_DocumentType]):
     def __init__(
         self,
         target: Union[
-            "MongoClient[_DocumentType]", "Database[_DocumentType]", "Collection[_DocumentType]"
```

Comment:
Is this allowed even though the MongoClient import is under `if TYPE_CHECKING:`?

Reply:
Yes, because it is converted to a string before it is interpreted.

Reply:
Neat!